app/templates/summary.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="data-container">
+  <table class="table table-bordered">
+    <thead>
+      <tr>
+        <th>
+            Name
+        </th>
+        <th>
+            Expences
+        </th>
+        <th>
+            Total Spent
+        </th>
+        <th>
+            Owes (Had To Pay)
+        </th>
+        <th>
+            Is Owned (Already Paid)
+        </th>
+        <th>
+            Balance
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+
+  {{#each person in controller.person}}
+    <tr>
+      <td>{{person.display_name}}</td>
+
+      <td>{{summary.expence-count person controller.expence}}</td>
+
+      <td>{{summary.total-spent person controller.expence}}</td>
+
+      <td>{{summary.total-owes person controller.expence}}</td>
+
+      <td>{{summary.total-owned person controller.expence}}</td>
+
+      <td>{{summary.total-balance person controller.expence}}</td>
```

Comment:
Overall good job in implementing the task.The summary is not very clear.
You should also show settlements across all transactions in (form of a graph or a table).
This new table/graph can contain how much each person needs to pay to the every other person to clear all the debts.
See this [https://github.com/ArtooTrills/Bounty-Web-Expenses#summary-sheet](url)